boards/arduino-atmega-common/include/periph_conf.h

```diff
@@ -139,6 +141,28 @@ extern "C" {
 #endif
 /** @} */
 
+/**
+ * @name ADC configuration
+ *
+ * The number of ADC channels of the atmega328p depends on the package:
+ *  - 6-channel 10-bit ADC in PDIP package
+ *  - 8-channel 10-bit ADC in TQFP and QFN/MLF package
+ * Arduino UNO / Duemilanove has thereby 6 channels. But only 5 channels can be
+ * used for ADC, because the pin of ADC5 emulate a software triggered interrupt.
+ *  -> boards -> arduino-atmega-common -> include -> board_common.h
+ *
+ * @{
+ */
+#ifdef CPU_ATMEGA328P
+#define ADC_NUMOF       (5)
+#endif
+#ifdef CPU_ATMEGA2560
+#define ADC_NUMOF       (16)
+#endif
+#ifdef CPU_ATMEGA1281
+#define ADC_NUMOF       (8)
+#endif
```

Comment:
Yes, but all periphery of these MCU's (e.g. SPI, UART) is defined in "boards/arduino-atmega-common/include/periph_conf.h". So I think, that the ADC defines can be remain there?